build.sbt

```diff
@@ -136,7 +136,8 @@ val commonSettings = Sonatype.sonatypeSettings ++ assemblySettings ++ Seq(
         if module.name.startsWith(name)
       } yield entry.data).toList.map((_, url(apiUrl)))
     }
-    val bootClasspath = System.getProperty("sun.boot.class.path").split(":").map(file(_))
+    val separator = if(System.getProperty("os.name").toLowerCase.contains("windows")) ";" else ":"
```

Comment:
You can also use `System.getProperty("path.separator")`

Reply:
Thanks, fixed!